src/models/templates.js

```diff
@@ -4,32 +4,111 @@ const templates = [
   {
     templateId: 1,
     templateName: "Appointment Reminder",
-    English: `Dear {{patientName}}, this message is to inform you of your upcoming appointment{{#description}}: {{description}}{{/description}}.
+    English: `
+      Dear {{patientName}}, this message is to inform you of your upcoming appointment{{#description}}: {{description}}{{/description}}.
```

Comment:
Just a minor thing, would be better to remove the `tab` characters preceding each line for templates. Just to reduce the number of chars from the SMS

Reply:
This applies to all other lang templates